public/app/features/admin/state/actions.ts

```diff
@@ -93,9 +93,9 @@ export function deleteUser(userId: number): ThunkResult<void> {
   };
 }
 
-export function updateUserPermissions(userId: number, isGrafanaAdmin: boolean): ThunkResult<void> {
+export function updateUserPermissions(userId: number, isServerAdmin: boolean): ThunkResult<void> {
   return async dispatch => {
-    const payload = { isGrafanaAdmin };
+    const payload = { isServerAdmin };
```

Comment:
Suggest to keep this backward compatible until v8:

```suggestion
    const payload = { isGrafanaAdmin: isServerAdmin };
```